chart/templates/deployment.yaml

```diff
@@ -30,6 +30,8 @@ spec:
               mountPath: /var/www/resources/elections
             - name: resources
               mountPath: /var/www/resources/resources
+            - name: breakages
+              mountPath: /var/www/resources/breakages
```

Comment:
I'm not sure why this isn't being flagged by the validation check, but you'll need to add a volume to the `volumes` section below.

Reply:
For reference, see this commit: https://github.com/backstage-technical-services/website/commit/682ff457afc2a52d617b32a5e2d72cc62c83ba05